### DISCUSSION TITLE
2024-09-25 Documentation Meeting

### DISCUSSION THREAD

**Discussion by glopesdev** [SainsburyWellcomeCentre/aeon_docs#25] [Meeting Agendas]
## Agenda

- All docs hosted in https://sainsburywellcomecentre.github.io/aeon_docs/ 
- Documentation use cases
- Software
- Python API
- Analysis [(Example notebooks)](https://lochhh.github.io/aeon_docs/user/index.html)
  - Python analysis
  - DJ Analysis
- Hardware / Firmware [(MWE)](https://sainsburywellcomecentre.github.io/aeon_docs/reference/hardware.html)
- Ephys
- Experiments

## Proposals

- https://github.com/SainsburyWellcomeCentre/aeon_docs/issues/24

**Comment by lochhh** [comment]
![image](https://github.com/user-attachments/assets/29fda49d-e111-4801-a9ce-b589c25fda32)
[Diataxis](https://diataxis.fr/)

**Comment by glopesdev** [comment]
- Docs Landing page:
  - I want to:
    - Build a new experiment with Aeon components
    - Analyse or understanding experiments that were run on Aeon
  - Browse Aeon functional modules (infrastructure docs)
  - Browse Aeon experiments (experiments docs)

- Minimal viable site for SfN

- Need to define functional modules for index (examples below)
  - Feeder
  - Nest
  - Tracking
  - Ephys
  - ...

**Comment by glopesdev** [comment]
For reference, additional examples of documentation websites using Bonsai which have been popping up:
  - https://allenneuraldynamics.github.io/Bonsai.AllenNeuralDynamics/
  - https://open-ephys.github.io/onix1-bonsai-docs/index.html
  - https://fchampalimaud.github.io/cf.bonsai/workflows/HarpExamples/BehaviorBoard/AnalogInput/AnalogInput.html
